jbpm-services/jbpm-services-api/src/main/java/org/jbpm/services/api/UserTaskService.java

```diff
@@ -494,6 +494,15 @@
      */
     Long saveContent(Long taskId, Map<String, Object> values);
     
+    /**
+     * Saves gives values as content of a task, applies to task output as input cannot be altered
+     * @param taskId
+     * @param values
```

Comment:
Please, add the missing parameter "userId" in the doc.

```suggestion
     * @param taskId
     * @param userId
     * @param values
```